migration_scripts/bulk_load/src/logger.clj

```diff
@@ -31,7 +31,7 @@
   ([] (->Logger *err*))
 
   ([writer]
-   (let [ch (chan)
+   (let [ch (chan (* 256 1024))
```

Comment:
It's the size of the buffer on this channel -- I was hitting some issues when there was a log output (more than 1000 messages queued up) where the logger would start spewing errors because its internal queue was filled up.